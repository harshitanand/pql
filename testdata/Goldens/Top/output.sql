SELECT * FROM "SourceFiles" ORDER BY "LineCount" DESC NULLS LAST LIMIT 3;
