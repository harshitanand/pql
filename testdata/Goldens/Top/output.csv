parser,parser_test.go,1108
parser,parser.go,878
parser,lex.go,681
